pkg/agent/cniserver/secondary.go

```diff
@@ -67,6 +67,13 @@ func (pc *podConfigurator) ConfigureSriovSecondaryInterface(
 
 // DeleteSriovSecondaryInterface deletes a SRIOV secondary interface.
 func (pc *podConfigurator) DeleteSriovSecondaryInterface(interfaceConfig *interfacestore.InterfaceConfig) error {
```

Comment:
Is it simpler to pass NetNS to this func as an argument, than adding a NetNS field to `InterfaceConfig` (finally we may want to save NetNS to interfaceStore, but I wonder for this PR do we really that)?

Reply:
Hi @jianjuns I went through the codes, both CNIServer Pod update events and Pod events may trigger syncPod() and do interface cleanup accordingly. If interfaces created for a previous Pod with the same Namespace/name are
not deleted yet, we still need to figure out the NetNS in the old stored interfaceConfig. So I would prefer to keep the NetNS in the interfaceConfig.